Documentation/components/k8s/developers.rst

```diff
@@ -7,37 +7,7 @@ Creating the Docker Images
 Docker 18.06 is needed to build the images, and the daemon has to be started with the `--experimental` flag.
 `See this issue to have more information <https://github.com/moby/moby/issues/32507>`_.
 
-You also need to ``export DOCKER_BUILDKIT=1`` before building the image
-
-::
-
-    docker build --build-arg KUBERNETES=true -t polycubenetwork/k8s-pod-network -f Dockerfile .
-    docker push polycubenetwork/k8s-pod-network
```

Comment:
@mauriciovasquezbernal my recommendation is to provide CLI in a generic way. 
E.g.:
```
docker build --build-arg KUBERNETES=true -t docker_repo/name:tag -f Dockerfile .
docker push docker_repo/name:tag
```

Reply:
I agree